lib/shoulda/matchers/active_record/have_rich_text_matcher.rb

```diff
@@ -0,0 +1,76 @@
+module Shoulda
+  module Matchers
+    module ActiveRecord
+      # The `have_rich_text` matcher tests usage of the
+      # `has_rich_text` macro.
+      #
+      # #### Example
+      #
+      #     class Post < ActiveRecord
+      #       has_rich_text :content
+      #     end
+      #
+      #     # RSpec
+      #     RSpec.describe Post, type: :model do
+      #       it { is_expected.to have_rich_text(:content) }
+      #     end
+      #
+      #     # Minitest (Shoulda)
+      #     class PostTest < ActiveSupport::TestCase
+      #       should have_rich_text(:content)
+      #     end
+      #
+      # @return [HaveRichText]
+      #
+
+      def have_rich_text(rich_text_attribute)
+        HaveRichText.new(rich_text_attribute)
+      end
+
+      # @private
+      class HaveRichText
+        def initialize(rich_text_attribute)
+          @rich_text_attribute = rich_text_attribute
+        end
+
+        def description
+          "have :#{rich_text_attribute} as a ActionText::RichText"
```

Comment:
This seems a bit clunky from an English perspective. What if we were to rename this to "have configured :#{rich_text_attribute} as a ActionText::RichText attribute"? Then the whole thing reads:

> Expected Post to have configured :content as a ActionText::RichText attribute